test/webtest/src/main/java/com/onfido/qa/websdk/model/Option.java

```diff
@@ -0,0 +1,32 @@
+package com.onfido.qa.websdk.model;
+
+import java.util.Objects;
+
+public class Option {
```

Comment:
Can we rename this to PoADocumentOption?

Reply:
It isn't just used for PoA, but for both document selectors. Therefore, I'll rename it to `DocumentOption`.